### DISCUSSION TITLE
Sys Reqs for set-up on Raspberry Pi / Rock Pi

### DISCUSSION THREAD

**Discussion by SilvanusP** [mbround18/valheim-docker#628] [Q&A] [answered]
Hi everybody,

with the Mistland Update soon being officially released I'm thinking about setting up a dedicated server for up to six players.

I googled the system requirements for hosting a dedicated server  and they seem quite high:

_Processor: 2.6 GHz Quad Core or similar.
Memory: 8 GB RAM.
Graphics: GeForce GTX 950 or Radeon HD 7970_
via Steam

_CPU Frequency: 2.8 GHz | 3.4 GHz+
RAM: 2 GB | 4 GB+_
[link](https://valheim.fandom.com/wiki/Valheim_Dedicated_Server)

At the same time (well 1.5 years ago) [this guy](https://ikarus.sg/valheim-server-rock-pi-x) did it with a Rock Pi with only 1.44GHz.

Does anyone have any recent experience with a Rasberry Pi / Rock Pi? How good does the processor actually have to be?
Does it need to be a Rock 5 with 2.4GHz or are 1.5GHz (Rasberry Pi 4) / 1.9GHz (Rock Pi X) enough?

(I found only one dicussion for this topic and i talks only about the amount of cores and RAM - which contradicts my search online)

**Comment by spannerman79** [comment]
Ask the author of https://github.com/mbround18/valheim-docker/discussions/531

Personally - Unity based games aren't known to be exactly efficient.  They eat 4GB RAM out of the gate so to speak.

**Comment by lyk4ndna** [comment, marked as answer]
Pretty much what @spannerman79 said. About a year and a half ago, I set up a server on a 4GB RockPI and it worked for awhile, but then kept running into performance issues with constant disconnects. 

We regularly had about 2 - 4 players on at a time but as the world grew, it became more unstable. As of right now as soon as it starts up it’s at 3.8 GB of RAM and I can play for a bit before it disconnects, then it crashes and burns. 

The RockPI isn’t gonna work as great as you want it to so it’s not worth it, better to spend the money on a cheap laptop computer with recommended specs if you want to host a server.

**Reply by SilvanusP** [reply]
Thank you!

Since Mistlands released yesterday and very unexpectedly for us, my friends wanna go the quick and dirty route and rent an online server. I'm very sceptical and when we tried the 2-day free trial of indifferentbroccoli.com one of five had a really bad experience and another one a semi-bad experience. 

In the long-term I will probably go for your recommendation, but for now I just hope we find a good server for the next month...

**Reply by mbround18** [reply]
@SilvanusP We use [eco.oeco.ovh.netvh.net](https://eco.ovhcloud.com/en/) for our servers & highly recommend. Just have peeps pitch in and it has more than enough performance to run multiple game servers.

**Reply by spannerman79** [reply]
There is also [Digital Ocean](https://www.digitalocean.com/products/droplets) too.

**Reply by lyk4ndna** [reply]
> Thank you!
> 
> Since Mistlands released yesterday and very unexpectedly for us, my friends wanna go the quick and dirty route and rent an online server. I'm very sceptical and when we tried the 2-day free trial of indifferentbroccoli.com one of five had a really bad experience and another one a semi-bad experience.
> 
> In the long-term I will probably go for your recommendation, but for now I just hope we find a good server for the next month...


I just looked up the Rock 5 and it seems to be adequate enough to run a Valheim server. Having 8GB of RAM and a quad processor. 

Just for reference I spun up a virtual box with specs matching the RockPi X (4GB RAM, quad core) and set it up via a minimal install of Debian 11. After spinning up the docker container it’s sitting at 2.46 GB RAM usage. 

I’ve noticed it will go up as more people join so with that being said I think you have wiggle room with the [Rock 5](https://www.okdo.com/us/p/okdo-rock-5-model-b-8gb-single-board-computer-rockchip-rk3588-arm-cortex-a76/). I still recommend the as bare minimum install of Linux without a desktop environment.
